### DISCUSSION TITLE
How to much time does it take to get the Student Developer pack activated once it has been submitted and what are the best benefits to avail.

### DISCUSSION THREAD

**Discussion by abhay702** [community/community#21051] [Other Feature Feedback, Questions, & Ideas]
What are some useful features of the students developers pack that must be availed and used to the fullest?

**Comment by FahdSeddik** [comment]
Hey! @abhay702 
They state that it is within 2 weeks , however, from my experience, I only got it after 1 day.
As for the features I think this [link](https://medium.com/backticks-tildes/github-student-developer-pack-demystified-32c3fb9260dc) might help.
